addons/pos_restaurant/static/src/app/services/pos_store.js

```diff
@@ -397,14 +399,16 @@ patch(PosStore.prototype, {
         try {
             this.loadingOrderState = true;
             const orders = await this.syncAllOrders({ throw: true });
```

Comment:
```suggestion
            const orders = (await this.syncAllOrders({ throw: true })) || [];
```

then restore the diff below.

Reply:
This part of the code was modified during the rebase.